### PR TITLE
 [sshd] Close SSH connections after 15 minutes of inactivity

### PR DESCRIPTION
**Pull request by jleveque**
**- What I did**

Configure sshd to close all SSH connetions after 15 minutes of inactivity.

**- How I did it**

Set `ClientAliveInterval` to `900` (900 seconds = 15 minutes) and `ClientAliveCountMax` to `0`
in /etc/ssh/sshd_config using augtool in build_debian.sh. In the process, I refactored the existing augtool command for sshd_config so as to add comments and empty lines to file for readability.

**- How to verify it**

1. Log into device via management port. Wait 15 minutes without sending a keystroke -- you should be automatically logged out.

### CODE REVIEW

**Comment by jleveque** [comment]
Retest vs please

File: build_debian.sh [hunk 1]
```diff
@@ -284,8 +284,28 @@ sudo chmod u+s $FILESYSTEM_ROOT/bin/ping{,6}
...
+rename /files/etc/ssh/sshd_config/EmptyLineHack ""
+set /files/etc/ssh/sshd_config/UseDNS no
+ins #comment before /files/etc/ssh/sshd_config/UseDNS
+set /files/etc/ssh/sshd_config/#comment[following-sibling::*[1][self::UseDNS]] "Disable hostname lookups"
+
+rm /files/etc/ssh/sshd_config/ClientAliveInterval
+rm /files/etc/ssh/sshd_config/ClientAliveCountMax
+touch /files/etc/ssh/sshd_config/EmptyLineHack
+rename /files/etc/ssh/sshd_config/EmptyLineHack ""
+set /files/etc/ssh/sshd_config/ClientAliveInterval 900
```
Comment (lguohan) [inline build_debian.sh]: i think we use curl -o to download the image. if the download exceed 15 minutes, does it break.

Comment (jleveque) [reply build_debian.sh]: I tested this using a timeout of 15 seconds and downloading a file which took over a minute. The curl operation keeps the session alive; the session closed 15 seconds after the curl process exited.

**Comment by toreanderson** [comment]
This is an abuse of the OpenSSH `ClientAlive*` settings, which are not intended as a generic idle timeout, see https://bugzilla.mindrot.org/show_bug.cgi?id=3172#c6 and https://bugzilla.mindrot.org/show_bug.cgi?id=3362#c2.

The fact that it did with `ClientAliveCountMax 0`, is actually a bug that was fixed in OpenSSH 8.2, see https://bugzilla.mindrot.org/show_bug.cgi?id=2627#c2. So the desired behaviour will stop working.

Fortunately, I might add. This is a pretty annoying «feature» – getting thrown out every 15 minutes is infuriating. Even one is not actually sending any keystrokes to the SSH session this does not mean that it is «inactive». One might be trying to debug some issue by passively monitoring streaming log output from `journalctl -f`, for example.

This pull request is IMHO missing a clear rationale or a «**Why** I did it» section. This is not the default behaviour in OpenSSH nor in any other mainstream Linux distribution as far as I know. Why deviate?

**Pull request merged**